projects/schema-form/src/lib/model/schemapreprocessor.ts

```diff
@@ -112,14 +114,14 @@ export class SchemaPreprocessor {
     fieldSchema.widget = widget;
   }
 
-  private static checkItems(jsonSchema, path) {
+  private static checkItems(jsonSchema: ISchema, path) {
     if (jsonSchema.items === undefined) {
       schemaError('No \'items\' property in array', path);
     }
   }
 
-  private static recursiveCheck(jsonSchema, path: string) {
-    if (jsonSchema.type === 'object') {
+  private static recursiveCheck(jsonSchema: ISchema, path: string) {
+    if (jsonSchema.type === FieldType.Object) {
```

Comment:
could this break something if we compare enums here vs strings??

Reply:
no problem